helper-cli/src/main/kotlin/commands/GenerateScopeExcludesCommand.kt

```diff
@@ -147,13 +152,18 @@ private fun getScopeExcludesForPackageManager(packageManagerName: String): List<
                 reason = ScopeExcludeReason.BUILD_DEPENDENCY_OF,
                 comment = "Packages for Kotlin compiler only."
             ),
+            ScopeExclude(
+                pattern = "kotlinNativeCompilerPluginClasspath",
+                reason = ScopeExcludeReason.BUILD_DEPENDENCY_OF,
+                comment = "Packages for Kotlin compiler only."
+            ),
             ScopeExclude(
                 pattern = "ktlint",
                 reason = ScopeExcludeReason.TEST_DEPENDENCY_OF,
                 comment = "Packages for code linting (testing) only."
             ),
             ScopeExclude(
-                pattern = "lintClassPath",
+                pattern = "lint.*",
```

Comment:
Which other lint* scopes were you actually facing?

Reply:
Seeing the following in our scans:

- lintChecks
- lintClassPath
- lintPublish